src/ts/ConvivaAnalytics.ts

```diff
@@ -261,7 +260,14 @@ export class ConvivaAnalytics {
       return;
     }
 
+    this.debugLog('[ ConvivaAnalytics ] report playback ended state');
     this.convivaVideoAnalytics.reportPlaybackEnded();
+
+    if (this.isAdBreak) {
+      this.debugLog('[ ConvivaAdAnalytics ] report ad skipped');
+      this.convivaAdAnalytics.reportAdSkipped();
```

Comment:
Not sure about it. This is inside `public endSession(): void {`. There are three options:
- Do nothing
- Report as skipped
- Report as ended

What do you the think is the best approach?

Reply:
Updated the docs in https://github.com/bitmovin/bitmovin-player-web-analytics-conviva/pull/104/commits/87ea855cdad922765f63a661c896910bbad97cd7.